### PR TITLE
Fixed things mentioned in issue #298

### PR DESCRIPTION
**Pull request by ShadowBB**
Fixed "Vídeo" typo
Fixed author name (Wilco Fiers)
Fixed Non-streaming link
Fixed 404 video's

+ Added rule type

Closes issue: #298 

## Guidance for the PR (pull request) creator

**When creating PR:**
- [ ] Make sure you requesting to **pull a issue/feature/bugfix branch** (right side) to the master branch (left side).

**After creating PR:**
- [ ] Add yourself (and co-authors) as "Assignees" for PR
- [ ] Add label to indicate if it's a `Rule`, `Definition` or `Chore` (more to the administrative side)
- [ ] Add relevant project (e.g. "Q3 2018 Status") to PR
- [ ] OPTIONAL: If you want anyone in particular to review your pull request, assign them as "Reviewers".
- [ ] Close the issue that the PR resolves (and make sure the issue is referenced in the top of this comment)

# How to Review And Approve
- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines. 
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.

### CODE REVIEW

File: _rules/SC1-2-2-video-has-audio-alternative.md [hunk 1]
```diff
@@ -53,7 +56,7 @@ There are no major accessibility support issues known for this rule.
 A video element with an associated track element that contains captions for all the audio.
 
 ```html
-<video src="../test-assets/perspective-video/perspective-video.mp4" controls>
+<video src="../test-assets/perspective-video/perspective-video.mp4.mp4" controls>
```
Comment (jkodu) [inline _rules/SC1-2-2-video-has-audio-alternative.md]: too many `.mp4`? extensions?

Comment (jkodu) [reply _rules/SC1-2-2-video-has-audio-alternative.md]: I would rather change the filename of the test asset

Comment (ShadowBB) [reply _rules/SC1-2-2-video-has-audio-alternative.md]: Did so. Thanks for the suggestion!

**Comment by jkodu** [comment]
@ShadowBB Upon, making the changes please dismiss my review. This will help notify me to review again.

**Comment by ShadowBB** [comment]
> 
> 
> @ShadowBB Upon, making the changes please dismiss my review. This will help notify me to review again.

Ah! I normally do this. I forgot this time. Thanks for the reminder and the approval!

**Pull request merged**